### ISSUE TITLE
MacOS Version

### ISSUE THREAD

**Issue by h4xor666** [NuclearDevelopers/Dichromate#18]
Hello there, friends. I come to you humbly asking what the chances are of making a version of this browser for MacOS (possibly M1?) if possible. If not, it's all good. Unfortunately, I am not experienced in such matters, so I am not sure how much of a help I could be. In any case, great browser, and I use it on my Windows machine.

**Comment by Douloureux** [comment]
Thank you for using Dichromate. 

Unfortunately, I do not own a Mac, ARM or Intel, so I am unable to provide MacOS builds of Dichromate. 

You can compile Dichromate [yourself on MacOS](https://chromium.googlesource.com/chromium/src/+/master/docs/mac_build_instructions.md). Just [apply the patches](https://github.com/NuclearDevelopers/Dichromate/blob/main/docs/Building.md#applying-the-dichromate-patches) from this repository before starting the compile. However, build process takes a very long time, and it would be a hassle keeping it up to date. Another issue is the fact that we don't have an automatic update system for MacOS, which is a security risk. I wouldn't recommend using your own builds unless you are sure you can keep it up to date.

I will leave this issue open, in case we are able to get MacOS builds going sometime in the future.
